api/src/main/java/org/openmrs/Person.java

```diff
@@ -59,6 +62,9 @@
 	@Field
 	private String gender;
 	
+
+	@Field(analyze = Analyze.YES)
+	@DateBridge(encoding = EncodingType.STRING, resolution = Resolution.YEAR)
```

Comment:
In any case, as a general rule, we'd actually want to store the resolution of the birth date as a DAY rather than a YEAR (even for the purposes of `getSimilarPeople()`, the resolution isn't quite a year, but a fuzzy "3 year period either side of this given year).

Reply:
I can see we do explicitly need a year. If we cannot construct a year query in Lucene, I think we should add a new calculated Lucene-only field of birthyear or something.